### ISSUE TITLE
v5.1.2.beta4 - beta11: Some devices are still green connected status when in fact, it's not connected already

### ISSUE THREAD

**Issue by shown19** [AdoPiSoft/Beta-Releases#29]
Here's my scenario, my PC is connected sa vendo po, it's weird lang na after I turned off the PC, still connected green status parin siya even hours have passed by. When I checked the traffic graph, may download activity siya pero yong upload activity niya is zero. I'm pretty sure walang mac cloning na naganap dahil I checked all the devices connected to my PC and there's no mac address or ip pertaining to my PC. If I delete the session, don lang po siya nawawala sa Activity filter pero once ibalik ko yong session, balik green status ulit. I even unplugged the cable to my PC pero green parin talaga.

P.S. nakasubsription po yong session with no expiration.

**Comment by darellep** [comment]
hello @shown19 , this issue has been fixed sa beta 14.

**Comment by darellep** [comment]
Hello @shown19 ,

We have a pre-release beta. You may want to update and confirm if this issue still exists.
https://github.com/darellep/adopisoft-beta-releases/releases

Thank you.

**Comment by shown19** [comment]
> Hello @shown19 ,
> 
> We have a pre-release beta. You may want to update and confirm if this issue still exists. https://github.com/darellep/adopisoft-beta-releases/releases
> 
> Thank you.

Updated to beta16, sadly yong PC ko is still green connected status parin and when checked sa network graph may activity sa download pero zero sa upload. Checked my APs for any possible mac spoofing that has same mac address or ip address pero wala naman. I don't know why pero weird lang that my PC is the only one has this issue or maybe diko lang napansin sa iba. I already tried deleting it and adding new session pero nasa active status talaga siya kahit naka power off na. Maybe deleting it permanently would fix it? Pero yong delete functionality sa system parang di permanent siya like inihide lang niya sa view devices, correct me if im wrong. Salamat. 

As for other issues, will test further lang muna with this update, so far I like the device activity history and merge feature. Update lang ako sa mga issues napost ko dito like duplicated device if nafix na talaga. Maraming salamat po.
